source/jormungandr/jormungandr/scenarios/helper_classes/fallback_durations.py

```diff
@@ -54,7 +54,8 @@
 
 # use dataclass when python3.7 is available
 DurationElement = namedtuple(
-    'DurationElement', ['duration', 'status', 'car_park', 'car_park_crowfly_duration', 'via_access_point']
+    'DurationElement',
+    ['duration', 'status', 'car_park', 'car_park_crowfly_duration', 'via_access_point', 'via_poi_access'],
```

Comment:
is it possible to rename `via_access_point` to `via_pt_access`?
```suggestion
    ['duration', 'status', 'car_park', 'car_park_crowfly_duration', 'via_pt_access', 'via_poi_access'],
```

Reply:
Done